DCCInspector-EX.ino

```diff
@@ -886,6 +884,26 @@ void DecodePacket(Print &output, int inputPacket, bool isDifferentPacket) {
               sbTemp.print(F(" F28-F21 "));
               sbTemp.print(dccPacket[inputPacket][pktByteCount - 1], BIN);
               break;
+            case 0B00011000:  // F29-F36 Function Control
+              sbTemp.print(F(" F36-F29 "));
+              sbTemp.print(dccPacket[inputPacket][pktByteCount - 1], BIN);
+              break;
```

Comment:
Is this where you are seeing oddities?

Reply:
I can't see anything wrong with this particular line.  Functions F29 and F36 are reported correctly.  No action (unless you let me know otherwise).